test-poc/pom.xml

```diff
@@ -35,6 +35,7 @@
     <modules>
         <module>examples</module>
         <module>showcase</module>
+        <module>providers</module>
```

Comment:
@vaceksimon I would rather put it under showcase, then having it standalone.

Reply:
like showcase/providers/pom.xml

Reply:
In that case, I need to create another module for the rest of the tests in the showcase module (currently only the UserTest). We plan to add more after the gap analysis, so this makes sense. WDYT?